glib/Marshaller.cs

```diff
@@ -23,12 +23,11 @@
 
 namespace GLib {
 	using System;
+	using System.Linq;
 	using System.Runtime.InteropServices;
 	
-	public class Marshaller {
+	public static class Marshaller {
```

Comment:
it'd probably be better to not break it then. There isn't a compelling reason to break this :)